File: Computer-Based-Test-Website/computer-based-test/app/Models/Operator.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Auth\Access\HandlesAuthorization;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\SoftDeletes;

class Operator extends Model
{
    use HasFactory, SoftDeletes;

    protected $guarded = ['id'];

    protected $fillable = [
        'nama_sekolah',
        'email',
        'password',
        'status_aktif',
        'durasi',
    ];
    
    public function user()
    {
        return $this->belongsTo(User::class);
    }

    public static $rules = [
        'status_aktif' => 'in:aktif,tidak aktif',
    ];
}

File: Computer-Based-Test-Website/API/models/Operator.go
package models

import "time"

type Operator struct {
    ID           uint      `gorm:"primaryKey" json:"id"`
    NamaSekolah  string    `json:"nama_sekolah"`
    Email        string    `json:"email"`
    Password     string    `json:"password"`
    StatusAktif  bool      `json:"status_aktif"`
    Durasi       int       `json:"durasi"`
    CreatedAt    time.Time `json:"created_at"`
    UpdatedAt    time.Time `json:"updated_at"`
}